### DISCUSSION TITLE
Logging mechanism

### DISCUSSION THREAD

**Discussion by cdot65** [opsmill/infrahub#3522] [Q&A] [answered]
How is logging of tasks handled today? I noted that the Helm Chart has Loki, but I'm unsure if that's being used in the fashion that I am thinking of.

**Comment by fatih-acar** [comment, marked as answer]
Hello,

Infrahub can run in `production` mode using the `INFRAHUB_PRODUCTION` parameter as described in the configuration reference: https://docs.infrahub.app/reference/configuration

This will enable JSON format for logs. You can take a look at a sample promtail configuration here that shows how to parse the logs (either in production or non-production mode): https://github.com/opsmill/infrahub/blob/develop/development/promtail/config.yml

The Helm Chart does not include anything specific to logging yet but you can still make use of the above points for your needs.
